### PR TITLE
Feedback window minor bugs fixed

### PR DESCRIPTION
**Pull request by bmourat**
Minor bugs fixed:
Bug with recycling images in tableview cells
Bug with attachment list not being cleared after sending message

### CODE REVIEW

File: Classes/Feedback/BITFeedbackMessageCellView.m [hunk 1]
```diff
@@ -211,6 +212,15 @@ - (void)drawRect:(NSRect)dirtyRect {
   [super drawRect:dirtyRect];
 }
 
+- (void)clearAllImageViews{
+  
+  // Since we are mutating array we need to go backwards
+  for(NSView *subview in [self.subviews reverseObjectEnumerator]){
+    if([subview isKindOfClass:[BITActivityIndicatorButton class]]){
```
Comment (lumaxis) [inline Classes/Feedback/BITFeedbackMessageCellView.m]: I think there are a couple of spaces missing here? 🙂

File: Classes/Feedback/BITFeedbackWindowController.m [hunk 2]
```diff
@@ -353,8 +353,10 @@ - (BOOL)canSendMessage {
 }
 
 - (IBAction)sendMessage:(id)sender {
-  [self.manager submitMessageWithText:[self.messageText string] andAttachments:self.attachments];
+  [self.manager submitMessageWithText:[self.messageText string] andAttachments:[[NSArray alloc] initWithArray:self.attachments]];
```
Comment (lumaxis) [inline Classes/Feedback/BITFeedbackWindowController.m]: If you want, a small syntax improvement could be to use `[NSArray arrayWithArray:self.attachments];` instead

Comment (bmourat) [inline Classes/Feedback/BITFeedbackWindowController.m]: Thanks :)

File: Classes/Feedback/BITFeedbackMessageCellView.m [hunk 3]
```diff
@@ -211,6 +212,15 @@ - (void)drawRect:(NSRect)dirtyRect {
   [super drawRect:dirtyRect];
 }
 
+- (void)clearAllImageViews{
+  
+  // Since we are mutating array we need to go backwards
```
Comment (TroubleMakerBen) [inline Classes/Feedback/BITFeedbackMessageCellView.m]: a)`mutating THE array`
b) which array, please be specific

**Pull request merged**